SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/Connection/SpatialWorkerConnectionTest.cpp

```diff
@@ -173,13 +182,16 @@ bool FFindWorkerResponseOfType::Update()
 {
 	bool bFoundOpOfExpectedType = false;
 	USpatialWorkerConnection* Connection = ConnectionManager->GetWorkerConnection();
-	Connection->Advance(0);
-	for (const auto& Op : Connection->GetWorkerMessages())
+	if (Connection)
```

Comment:
Hm, I probably don't have enough context here - but why do we now wrap everything with `if(Connection)`?
Seems to me that we now wouldn't catch if the connection setup failed.

Reply:
So yeah, this was to prevent an editor crashing failure in the initial iterations of v15. Essentially these tests were broken, if the first test failed, the test that establishes the WorkerConnection, these next tests will attempt to use the nullptr worker connection with
```
	if (Connection)Connection->SendDeleteEntityRequest(EntityId, RETRY_UNTIL_COMPLETE);
```
or similar. 
Wrapping in `if(Connection)` prevents the bad usages and the tests just time out in a couple seconds. I've updated this now to check 
```
	if (Connection == nullptr)
	{
		return false;
	}
```
instead now.